pkg/gofr/websocket/websocket_test.go

```diff
@@ -130,6 +130,7 @@ func Test_UnimplementedMethods(t *testing.T) {
 	assert.Equal(t, "", conn.PathParam("test"))
 	assert.Equal(t, "", conn.HostName())
 	assert.NotNil(t, "", conn.Context())
```

Comment:
This one is stupid. It's not related to your PR, but please fix it


```suggestion
 	assert.NotNil(t, conn.Context())
```

Reply:
Sure done